pkgs/tools/networking/croc/test-local-relay.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, croc }:
+
+stdenv.mkDerivation {
+  name = "croc-test-local-relay";
+  meta.timeout = 10;
```

```suggestion
  meta.timeout = 300;
```

I'm sure 10s is enough for a slow machine but I'd rather add some margin to make sure no false negatives occur, even on a heavily loaded slow machine.

---

Changed.